gateway-ha/src/main/java/io/trino/gateway/ha/router/TrinoQueryProperties.java

```diff
@@ -260,27 +274,30 @@ private String decodePreparedStatementFromHeader(String headerValue)
         return new String(preparedStatement, UTF_8);
     }
 
-    private void getNames(Node node, ImmutableSet.Builder<QualifiedName> tableBuilder,
+    private void getNames(
+            Node node,
+            List<QualifiedName> allQualifiedNames,
```

Comment:
I don't understand the difference between `tableBuilder` and `allQualifiedNames`. What's the motivation of this renaming?